lib/project_types/script/messages/messages.rb

```diff
@@ -18,8 +18,12 @@ module Messages
           invalid_context_cause: "Your .shopify-cli.yml file is not correct.",
           invalid_context_help: "See https://help.shopify.com",
 
+          invalid_script_name_cause: "Invalid script name.",
+          invalid_script_name_help: "Replace or remove unsupported characters. Valid characters "\
+                                    "are numbers, letters, spaces, hyphens, or underscores.",
```

Comment:
We saying spaces are allowed, but it looks like we implicitly convert them to underscores?

Reply:
True good catch, I just copy pasted this so I forgot to check. Thanks!